.eslintrc.cjs

```diff
@@ -167,6 +167,8 @@ module.exports = {
         'jest/no-focused-tests': 'error',
         'jest/no-identical-title': 'error',
         'jest/prefer-to-be': 'error',
+        'jest/prefer-to-contain': 'error',
```

Comment:
`jest/prefer-to-contain` did not find any errors. Added it just for completeness.